src/wazuh_modules/wm_ciscat.c

```diff
@@ -1499,7 +1509,27 @@ cJSON *wm_ciscat_dump(const wm_ciscat * ciscat) {
 
 
 // Destroy data
+#ifdef WIN32
+DWORD WINAPI wm_ciscat_destroy(void *ciscat) {
+
+    wm_ciscat_eval *cur_eval;
+    wm_ciscat_eval *next_eval;
+    wm_ciscat *ptr_ciscat = (wm_ciscat *)ciscat;
+
+    // Delete evals
 
+    for (cur_eval = ptr_ciscat->evals; cur_eval; cur_eval = next_eval) {
+
+        next_eval = cur_eval->next;
+        free(cur_eval->path);
+        free(cur_eval->profile);
+        free(cur_eval);
+    }
+
+    free(ptr_ciscat);
+    return 0;
```

Comment:
this could be summarized with a #ifdef WIN32 on the return to avoid the entire duplication of the function.